### PR TITLE
XW-3980 | fallback languages for wiki recommendations in MCF

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-3980
@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/Recirculation/WikiRecommendations.class.php [hunk 1]
```diff
@@ -619,6 +620,21 @@ public static function getRecommendations( $contentLanguage ) {
 		return $recommendations;
 	}
 
+	private static function fallbackToSupportedLanguages( $language ) {
+		switch ( $language ) {
```
Comment (WikiaTech) [inline extensions/wikia/Recirculation/WikiRecommendations.class.php]: MAJOR Add a "case default" clause to this "switch" statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)

File: extensions/wikia/Recirculation/WikiRecommendations.class.php [hunk 2]
```diff
@@ -619,6 +620,21 @@ public static function getRecommendations( $contentLanguage ) {
 		return $recommendations;
 	}
 
+	private static function fallbackToSupportedLanguages( $language ) {
```
Comment (WikiaTech) [inline extensions/wikia/Recirculation/WikiRecommendations.class.php:623]: MAJOR Reduce the number of returns of this function 4, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/Recirculation/WikiRecommendations.class.php (hunk 2 above)
Comment (xkxd) [inline extensions/wikia/Recirculation/WikiRecommendations.class.php:623]: i'd name this `normalizeSupportedLanguages`, but it's up 2 u

**Review by xkxd — APPROVED** [review]

**Review by JamesHawking — APPROVED** [review]

**Pull request merged**